source/FreeRTOS_UDP_IPv4.c

```diff
@@ -279,7 +262,11 @@ void vProcessGeneratedUDPPacket_IPv4( NetworkBufferDescriptor_t * const pxNetwor
         }
     }
 
-    if( eReturned != eCantSendPacket )
+    if( xLostBuffer == pdTRUE )
```

Comment:
In the case of `ipIPv4_FRAME_TYPE`, `xLostBuffer` doesn't seem to be set

Reply:
Currently `xStartAddressResolution` works differently for ND and ARP because `vNDSendNeighbourSolicitation` passes control of the packets (can't use FreeRTOS_OutputARPRequest because that works differently and creates new buffers). I was going to make an internal vOutputARPRequest that works like the ND so that `xStartAddressResolution` work similarly for both but I will save that for later.